src/v/redpanda/application.cc

```diff
@@ -206,6 +206,17 @@ void application::configure_admin_server() {
           })
           .get0();
     }
+    if (conf.dashboard_dir()) {
+        _admin
+          .invoke_on_all([](ss::http_server& server) {
+              server._routes.add(
+                ss::httpd::operation_type::GET,
+                ss::httpd::url("/"),
+                new ss::httpd::directory_handler(
+                  *config::shard_local_cfg().dashboard_dir()));
+          })
+          .get0();
```

Comment:
Very minor nit (not blocking): As of Seastar API level 6, future is no longer variadic (a tuple).
```suggestion
          .get();
```

Reply:
thank you for this. We should do a program-wide analysis and fix them all ?